src/postings/postings_writer.rs

```diff
@@ -36,7 +36,7 @@ fn posting_from_field_entry<'a>(
                 }
             })
             .unwrap_or_else(|| SpecializedPostingsWriter::<NothingRecorder>::new_boxed(heap)),
-        FieldType::U64(_) | FieldType::I64(_) | FieldType::HierarchicalFacet => {
+        FieldType::U64(_) | FieldType::I64(_) | FieldType::HierarchicalFacet | FieldType::Bytes => {
```

Comment:
`FieldType::Bytes` cannot be indexed at all. So you probably want a specific case.
`NothingRecorder` is a bit misleading here. It deos not "not index" the field. It indexes it with no extra informaiton (like docfreq or position)

Reply:
I'm not sure what the right solution is here.

One option is to return on `Option<Box<PostingsWriter + 'a>>` and never create a postings writer for the field. I don't know enough to know if this will cause downstream blowups.

Another option is to return a `ReallyDoesNothingRecorder` (obviously not the real name :smiley:).

Reply:
I think you are right. The whole thing will stay ugly as long as it does not under go a rather large refactoring.

Reply:
Can you just add the following comment :
// `FieldType::Bytes` cannot actually be indexed.
// TODO fix during the indexer refactoring described in #276

Reply:
Done